desktop-widgets/mainwindow.cpp

```diff
@@ -53,9 +53,9 @@
 #include "profile-widget/profilewidget2.h"
 
 #ifndef NO_PRINTING
-#include <QPrintDialog>
-#include <QBuffer>
-#include "desktop-widgets/printdialog.h"
+    #include <QPrintDialog>
+    #include <QBuffer>
+    #include "desktop-widgets/printdialog.h"
```

Comment:
I'm not sure I like that.
But if you do it, do it with a **TAB** :-)

Reply:
I blame my editor configuration. I like this because it makes sense: it's a branch inside of the if, even if it's an #if.

Reply:
Generally, I think perfectly consistent style is a bit overrated. As long as it's readable one can tolerate a bit of individuality.

But if you change it here, perhaps change it globally, so that it doesn't stand out. PS another style to consider might be:
```
#ifndef ...
#       define ...
#       define ...
#endif
```